org.opent2t.sample.hub.superpopular/com.smartthings.hub/js/groovy/SmartApp.groovy

```diff
@@ -102,25 +102,28 @@ def installed() {
 
 def updated() {
 	log.debug "Updating with settings: ${settings}"
-	if(state.deviceSubscriptionMap == null){
+	
+    //Initialize state variables if didn't exist.
+    if( state.deviceSubscriptionMap == null ){
 		state.deviceSubscriptionMap = [:]
-		log.debug "deviceSubscriptionMap created."
```

Comment:
these log statements are helpful in the debugging process, please don't delete them.  We used them last week.